app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -188,13 +188,59 @@
                                     :id                => record.id,
                                     :prefix            => "amqp",
                                     :basic_info_needed => true}
+          .col-md-12{"ng-if" => "#{ng_model}" == "emsCommonModel" && "#{ng_model}.emstype == 'vmware_cloud'"}
+            %div{"ng-if" => "emsCommonModel.event_stream_selection != 'amqp'"}
+              %div{"ng-init" => "emsCommonModel.event_stream_selection = 'none'"}
```

Comment:
I see that it works, I just wonder if it would be possible to make it resemble the OpenStack completely?